root/static/scripts/common/utility/formatSetlist.js

```diff
@@ -25,18 +25,22 @@ function setlistLink(entityType, entityGid, content) {
 const linkRegExp =
   /^\[([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})(?:\|([^\]]+))?\]/i;
 
-function formatSetlistArtist(entityGid, content) {
+function formatSetlistArtist(content, entityGid = undefined) {
```

Comment:
Parameters always default to undefined if you don't pass them, so `entityGid = undefined` doesn't actually do anything, but if you want to indicate that it's optional you can make that explicit with Flow:
```suggestion
function formatSetlistArtist(content: string, entityGid?: string) {
```